AasxFileServerServiceSpecification/V3.1_SSP-001.yaml

```diff
@@ -183,8 +183,6 @@ paths:
           $ref: 'https://api.swaggerhub.com/domains/Plattform_i40/Part2-API-Schemas/V3.1.0#/components/responses/unauthorized'
         '403':
           $ref: 'https://api.swaggerhub.com/domains/Plattform_i40/Part2-API-Schemas/V3.1.0#/components/responses/forbidden'
-        '404':
```

Comment:
When a new resource is created, RFC 9110 recommends returning a 201 Created status and ideally the location point to the new resource. Location is from my point of view not so critical as long the id or better the returned updated Json serialization of the object including an identifier to access the newly created object, e.g. a shell is in place. The information that it was created and was not an update is for me more a mandatory information or better could be for use cases really crucial. Extending that it is more me a breaking change.

Reply:
As in the API working group discussed, 201 will be used as defined in the RFC 9110. Related to the I'm fine and resolve the conversation.